src/client/app/components/about.ts

```diff
@@ -1,6 +1,10 @@
-import { Component, Input, ChangeDetectionStrategy, Output, EventEmitter, AfterViewInit } from '@angular/core';
+import { Component, Input, ChangeDetectionStrategy, Output, EventEmitter, AfterViewInit, ChangeDetectorRef } from '@angular/core';
 import { environment, storageSize, storage } from '../helpers';
 import {Strings, getAvailableLanguages, getDisplayLanguage, setDisplayLanguage } from '../strings';
+import { UIEffects } from '../effects/ui';
+const PLAYGROUND_ORIGIN = 'playground_origin_environment_url';
```

Comment:
Which is why we have an `environment.ts`. The idea is that we use webpack to inject a constant based on the env that it was built for. So you cant access `config/env.config` cause that's outside the projects build folder.

You can access the constants from https://github.com/OfficeDev/script-lab/blob/master/src/client/app/helpers/environment.ts where `PLAYGROUND` is injected in via https://github.com/OfficeDev/script-lab/blob/master/config/webpack.common.js#L69 which gets the build info and config from https://github.com/OfficeDev/script-lab/blob/master/config/webpack.common.js#L9